src/aof.c

```diff
@@ -1112,7 +1112,7 @@ int loadSingleAppendOnlyFile(char *filename) {
     long loops = 0;
     off_t valid_up_to = 0; /* Offset of latest well-formed command loaded. */
     off_t valid_before_multi = 0; /* Offset before MULTI command loaded. */
-    off_t base_size = 0, progress_size = 0;
+    off_t last_progress_report_size = 0, progress_size = 0;
```

Comment:
maybe it would be cleaner if `progress_size` will be a local variable declared when setting it?
and / or call it `progress_delta`?

Reply:
done